untitled1/src/main/com/aca/homework/week8/carrepository/Car.java

```diff
@@ -0,0 +1,45 @@
+package com.aca.homework.week8.carrepository;
+
+public class Car {
+    private String vin;
+    private int year;
+    private int ownersCount;
+
+    public Car(String vin, int year, int ownersCount) {
+        this.vin = vin;
+        this.year = year;
+        this.ownersCount = ownersCount;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || o.getClass() != Car.class) return false;
+        Car car = (Car) o;
+        return year == car.year && ownersCount == car.ownersCount && vin.equals(car.vin);
+    }
+
+    public int getOwnersCount() {
+        return ownersCount;
+    }
+
+    public String getVin() {
+        return vin;
+    }
+
+    public int getYear() {
+        return year;
+    }
+
+    public void setOwnersCount(int ownersCount) {
+        this.ownersCount = ownersCount;
+    }
+
+    public void setYear(int year) {
+        this.year = year;
+    }
+
+    public static Car carOf(String vin, int year, int ownersCount) {
```

Comment:
you do not need this static factory because it is not better than the constructor